dbt/models/default/schema.yml

```diff
@@ -31,15 +31,15 @@ models:
             - pin
             - year
           config:
-            error_if: ">280655"
+            error_if: ">280659"
```

Comment:
I have confirmed by running these tests locally that these changes represent additional malformed data that has been added to our source data since this PR opened. The fact that this happened twice during the ~week that this PR was open is an indication to me that I may want to pivot soon to focusing on resolving these data problems, since they're going to become annoying quite quickly once we're running these tests on every PR.

Reply:
**suggestion (non-blocking):** Some of these are likely just under-specified tests, but I agree it's going to be a problem. Grab @wrridgeway this week for a review/debugging session of the current tests. He knows more than anyone but Mirella about the horrors of our changing data.

Reply:
I scheduled some time tomorrow afternoon! In the meantime, I had to bump the thresholds yet again in 672a7d47628fcd1f868dbc07cd8ae8e8c560efff ☹️